modules/apps/document-library/document-library-api/src/main/java/com/liferay/document/library/preview/exception/DLPreviewSizeException.java

```diff
@@ -13,18 +14,53 @@
 public class DLPreviewSizeException extends PortalException {
 
 	public DLPreviewSizeException() {
+		this(0);
+	}
+
+	public DLPreviewSizeException(long maxSize) {
+		_maxSize = maxSize;
+	}
+
+	public DLPreviewSizeException(long maxSize, Throwable throwable) {
+		super(throwable);
+
+		_maxSize = maxSize;
 	}
 
 	public DLPreviewSizeException(String msg) {
+		this(msg, 0);
+	}
+
+	public DLPreviewSizeException(String msg, long maxSize) {
```

Comment:
Parameters need sorting?

Reply:
on this case, the msg of the exception always goes fist.
golden rule on my mind